docs/decisions/0007-dotnet-project-structure.md

```diff
@@ -0,0 +1,200 @@
+---
+
+# These are optional elements. Feel free to remove any of them
+
+status: proposed
+date: {YYYY-MM-DD when the decision was last updated}
+deciders: {list everyone involved in the decision}
+consulted: {list everyone whose opinions are sought (typically subject-matter experts); and with whom there is a two-way communication}
+informed: {list everyone who is kept up-to-date on progress; and with whom there is a one-way communication}
+---
+
+# DotNet Project Structure for 1.0 Release
+
+## Context and Problem Statement
+
+{Describe the context and problem statement, e.g., in free form using two to three sentences or in the form of an illustrative story.
+ You may want to articulate the problem in form of a question and add links to collaboration boards or issue management systems.}
+
+### Current Project Structure
+
+```text
+SK-dotnet
+├── samples/
+└── src/
```

Comment:
Curious if folks have thoughts of breaking this into `src/` and `test/`

Reply:
@lemillermicrosoft One of the options being considered no includes that. I'm hoping we do this as it will force is to divide our tests.

Reply:
This change is part of the final ADR revision